src/app/quiz/[quizId]/@detail/page.tsx

```diff
@@ -10,20 +16,35 @@ type Props = {
   };
 };
 
-async function DetailPage({ params: { quizId } }: Props) {
+function DetailPage({ params: { quizId } }: Props) {
+  const { submitQuiz } = useSubmitQuizMutation(quizId);
+  const quizDetail = useGetQuizDetailQuery(quizId);
+  if (quizDetail === null) {
+    return null;
+  }
```

Comment:
```suggestion
  const { data: quizDetail = DEFAULT_QUIZ_DETAIL }= useGetQuizDetailQuery(quizId);
```
을 하면 null 처리안하셔도 되긴합니다

Reply:
옹 이건 제가 잘 모르고 있던 부분인데 혹시 뭐라고 검색해야 나올까요?

Reply:
data값이 undefined로 정의되어 있어서 그냥 타입정의를 임의로 해준거에요 default넣어서!

Reply:
이 부분은 시도해보려다가 값이 넘 많아서... null처리 하는 방식 유지하겠습니다..!!